ksqldb-functional-tests/src/test/resources/sql-tests/query-upgrades/rowpartition-rowoffset.sql

```diff
@@ -32,4 +32,30 @@ ASSERT VALUES b (id, col1) VALUES (1, 0);
 
 SET 'ksql.rowpartition.rowoffset.enabled' = 'true';
 
-CREATE OR REPLACE TABLE b AS SELECT id, col1 FROM a WHERE col1 > ROWOFFSET EMIT CHANGES;
\ No newline at end of file
+CREATE OR REPLACE TABLE b AS SELECT id, col1 FROM a WHERE col1 > ROWOFFSET EMIT CHANGES;
+
+----------------------------------------------------------------------------------------------------
+--@test: should fail on querying stream with user columns named the same as new pseudocolumns
+--@expected.error: io.confluent.ksql.util.KsqlException
+--@expected.message: You cannot query a stream or table that has user columns with the same name as new pseudocolumns.
+----------------------------------------------------------------------------------------------------
+SET 'ksql.rowpartition.rowoffset.enabled' = 'false';
```

Comment:
Very clever use of this testing framework! Unfortunately I don't think we'll be able to keep these tests after the feature flag is removed, but let's leave them for now. (Extra motivation to add the unit tests described above.)